packages/eds-core-react/src/components/Datepicker/calendars/Calendar.tsx

```diff
@@ -0,0 +1,111 @@
+import { useCalendar, useLocale } from 'react-aria'
+import { forwardRef, ReactNode, RefObject, useEffect, useState } from 'react'
+import { CalendarStateOptions, useCalendarState } from '@react-stately/calendar'
+
+import { CalendarGrid } from './CalendarGrid'
+import { CalendarHeader } from './CalendarHeader'
+import { HeaderProps } from '../props'
+import { DatePickerState } from '@react-stately/datepicker'
+import {
+  CalendarDate,
+  CalendarDateTime,
+  createCalendar,
+  toCalendarDate,
+} from '@internationalized/date'
+import { Popover } from '../../Popover'
+
+/**
+ * Calendar is the inline calendar picker used in {@link DatePicker}.
+ * It only allows selecting a single date
+ */
+export const Calendar = forwardRef(
+  (
+    {
+      Header,
+      footer,
+      state,
+      ...props
+    }: {
+      /**
+       * Custom header component
+       */
+      Header?: (props: HeaderProps) => ReactNode
+      /**
+       * Custom footer component
+       */
+      footer?: ReactNode
```

Comment:
I would recommend using the same type as the Header to be consistent and provide context if we need it.